doc/lifecycle.md

```diff
@@ -0,0 +1,24 @@
+[back](../README.md)
+
+# Lifecycle
+
+Before starting a new development phase, we usually update the frameworks and dependencies first.
```

Comment:
Ich frage mich noch, ob wir dies hier so beschreiben wollen/sollen, oder ob es diesen Satz gar nicht braucht?

Vielleicht würde ich das ganze noch in eine Subsection «Update Dependencies» moven und noch eine weitere Subsection «Uncluttering» hinzufügen, wo wir `npm run unsed` (Knip) beschreiben? Wir haben das schon im Setup drin, aber ich fänd es hier eigentlich fast passender. Was meinst du?

Reply:
Weiter könnten wir vielleicht bei «Angular» und bei «Other dependencies» am Schluss einen Punkt hinzufügen:

```
- Make sure that all tests and linting are successful. If appropriate, test certain components/aspects manually.
```